meetup/models.py

```diff
@@ -1,3 +1,62 @@
+from uuid import uuid4
+
+from django.contrib.auth.models import User
 from django.db import models
 
-# Create your models here.
+
+class Venue(models.Model):
+    """This model is for the venue where be held"""
+    name = models.CharField(max_length=50)
+    location = models.CharField(max_length=255, null=True)
+    description = models.TextField(null=True)
+    latitude = models.FloatField()
+    longitude = models.FloatField()
+
+
+class ProgramCategory(models.Model):
+    """This model is for the category of program"""
+    name = models.CharField(max_length=100)
+    slug = models.SlugField(max_length=100, unique=True)
+
+
+class Speaker(models.Model):
+    """This model is for that user related by speaker at program"""
+    user = models.ForeignKey(User)
+
+
+class Program(models.Model):
+    """This model is for that lecture or event by speaker(s)"""
+    # There should be difficulty and language information in Program?
+    title = models.CharField(max_length=255)
+    brief = models.TextField(null=True, verbose_name='simple information')
+    description = models.TextField(null=True)
+    speakers = models.ManyToManyField(Speaker)
+    category = models.ForeignKey(ProgramCategory, verbose_name='The category which this program is belonged')
+    slide_url = models.CharField(max_length=255, null=True, verbose_name='A slide url')
```

Comment:
`URLField` is more fit here I think. (https://docs.djangoproject.com/ko/1.11/ref/models/fields/#urlfield)